connector/connectorRunner.go

```diff
@@ -60,10 +62,31 @@ func (cr *connectorRunner) Run() error {
 		return err
 	}
 
+	var (
+		server *factory.GRPCServer
+		writer process.Writer
+	)
+	if cr.config.GRPC.Enable {
+		writer = &fakeWriter{}
+		handler, err := process.NewGRPCBlocksHandler(outportBlocksPool, dataAggregator)
+		if err != nil {
+			return fmt.Errorf("couldn't create grpc blocks handler, error: %w", err)
+		}
+		server = factory.NewServer(cr.config.GRPC, handler)
+
+		go func() {
+			if err := server.Start(); err != nil {
+				log.Error("couldn't start grpc server", "error", err)
+			}
+		}()
```

Comment:
i suggest to move this part into the componet itself

Reply:
moved.